README.md

```diff
@@ -10,22 +10,103 @@ Inside the Arsenal Library you have different options to make amazing and quick
 ###Window Transitions
 
 Transitions:
--Fade
--Slide
--Explode
--Slide Explosion (Custom from Library)
+
+- Fade
+- Slide
+- Explode
+- Slide Explosion (Custom from Library)
 
 ###Animations
 
 Resource Animations:
--Fade In
--Fade Out
--Slide Left
--Slide Right
--Rotate
--Custom (Application Resource)
 
-##Usage
+- Fade In
+- Fade Out
+- Slide Left
+- Slide Right
+- Rotate
+- Custom (Application Resource)
+
+##In Use
+
+For Window Transitions simply call:
+
+```java
+  // Here we are setting the Enter Transition
+  AnimationArsenal.setEnterTransition(getWindow(), AnimationArsenal
+          .getFadeTransition(new Transition.TransitionListener()
+          {
+              @Override
+              public void onTransitionEnd(Transition transition)
+              {
+                  // In here you can start other animations at the end of the enter transition
+                  AnimationArsenal.circularReveal(myView,
+                          getApplicationContext(), AnimationArsenal.RevealGravity.CENTER);
+              }
+          }, Fade.IN));
+
+  // As well you can have total control of the exit transition
```

Comment:
`As well you can have total control...` Does not sound correct, probably it should be `You can also have total control of exit transition` ?